packages/client/serviceWorker/sw.ts

```diff
@@ -73,7 +74,9 @@ const onFetch = async (event: FetchEvent) => {
       return cachedRes
     }
     try {
-      const networkRes = await fetch(request)
+      // request.mode could be 'no-cors'
+      // By fetching the URL without specifying the mode the response will not be opaque
+      const networkRes = await fetch(request.url)
```

Comment:
want to get your thoughts on this! it's the same thing we do up above for all the assets that go into the static cache

Reply:
I have some trouble testing a production build locally, also on master. Would be nice to get this PR running on a on-demand k8s environment, but we could also test on staging.
It seems to  fetch with `cors` for the static content, so I think you're right, it should work for the dynamic ones as well. It probably fails if we fetch stuff from 3rd parties.

Reply:
oh! that's no good. what's the error you see? I can help debug!

my checklist for testing prod build locally:
- close all but 1 tab of parabol, unregister service worker, hard refresh (cmd + shift + r on mac)
- `yarn build && yarn predeploy && yarn dev` making sure `CDN_BASE_URL` is undefined


to test 3rd party stuff I turned on the new checkout flow & went to the org page so stripe would fetch its stuff. It works! We could scope it to just our assets that we serve from our CDN if you think that's safer. Your call!

<img width="1077" alt="Screenshot 2023-12-04 at 3 29 04 PM" src="https://github.com/ParabolInc/parabol/assets/5514175/7f0d78e4-bfae-431f-b8da-390d9ac34182">

Reply:
I'm running `yarn build && yarn predeploy && yarn start` with the CDN set to action-files.parabol.co/development. It used to work a month ago. It appears to be an infrastructure issue. I used to deploy the files to action-files/development, but now I cannot access the data there anymore.
I cannot successfully run `yarn predeploy` without CDN set, also I wonder how accurate testing would be without an external server.

Reply:
> I cannot successfully run yarn predeploy without CDN set,

what's the error you're seeing?




FWIW here's the output i get when using the following & building locally

```
CDN_BASE_URL='//action-files.parabol.co/development'
FILE_STORE_PROVIDER='s3'
```

<img width="1268" alt="Screenshot 2023-12-07 at 3 05 03 PM" src="https://github.com/ParabolInc/parabol/assets/5514175/343f34fe-36d7-4731-8404-129c1b22f4e2">

Reply:
switched to only cache things coming from parabol or the parabol CDN ✅ 
true, we don't rotate the dynamic cache! I think that's probably OK, there's no TTL available so it's probably better to flush on version than let it grow